boards/riscv/tlsr9518adk80d/tlsr9518adk80d_defconfig

```diff
@@ -13,6 +13,7 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_SERIAL=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
+CONFIG_2ND_LVL_ISR_TBL_OFFSET=12
```

Comment:
Move to SOC

Reply:
removed from here, already defined in `zephyr/soc/riscv/riscv-privilege/telink_b91/Kconfig.defconfig.series`